contributors/design-proposals/storage/postpone-pv-deletion.md

```diff
@@ -0,0 +1,73 @@
+# Postpone deletion of a Persistent Volume if it is bound by a PVC
+
+Status: Pending
+
+Version: Alpha
+
+Implementation Owner: NickrenREN@ 
+
+## Motivation
+
+User can delete a Persistent Volume (PV) that is being used by a PVC.  It may result in data loss.
```

Comment:
Nit: actually, only an admin can delete a PV. So this feature is for admins and doesn't affect ordinary users.

Reply:
yeah, will update this, thanks